pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
Honestly, I would prefer that artman just didn't try to put files directly into `googleapis`.

I do frequently edit the GAPIC config when I'm starting on a new API, since there's still significant manual work required sometimes, and I don't always remember `git add` it right after I make my edits. I then run the `GapicClient` artman pipeline -- I can imagine a frustrating situation where I accidentally run the `GapicConfig` pipeline instead and lose all my changes.